data/state.py

```diff
@@ -2,8 +2,24 @@
 
 
 class State(object):
-  UNUSED_STATUS = 0
-  PENDING = 1
-  FINISHED = 2
-  FAILED = 3
-  ABANDONED = 4
+  UNUSED_STATUS = 1
```

Comment:
why updating from 1 to 0 ?

Reply:
state = 0 is the argument that the frontend sends when no state filter has been selected, so I didn't want this to be confused with UNUSED_STATE which is an actual state type that can be filtered for.